pom.xml

```diff
@@ -3,17 +3,17 @@
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>1.509</version>
+		<version>1.557</version>
```

Comment:
with `1.557` some test cases fail, where there are no issues with version `1.509`.
btw. you can see this also in the failed builds for this PR: https://jenkins.ci.cloudbees.com/job/plugins/job/repository-connector-plugin/40/ (which is also indicated on the PR it self)